### DISCUSSION TITLE
Security Vulnerability Disclosure

### DISCUSSION THREAD

**Discussion by arunstar** [milvus-io/milvus#29411] [Q&A and General discussion]
Does `milvus` participate in any vulnerability disclosure programs? I believe I've found a security issue.

**Comment by yhmo** [comment]
We have set up a security policy for milvus project: https://github.com/milvus-io/milvus/security   (only the project owners can see the vulnerability warnings). The policy automatically submits pr to upgrade third parties for us.
You also can raise an issue if you find any security risks.

**Reply by arunstar** [reply]
Hey, sorry for the late reply. I couldn't see the **"Report a vulnerability"** option in security page. I believe you need to enable **"Private vulnerability reporting"** option in settings.

**Reply by yhmo** [reply]
We are planning to create a special mailbox for vulnerability reports. Please wait.
